assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -755,31 +755,7 @@ def get_token(self):
         pass
 
 
-class _ManagedIdentityAPITokenManager(_APITokenManager):
-    def __init__(
-        self,
-        *,
-        endpoint_type,
-        auth_header,
-        token_scope,
-        **kwargs,
-    ):
-        super().__init__(endpoint_type=endpoint_type, auth_header=auth_header)
-        self.token_scope = token_scope
-
-    def get_token(self):
-        if (
-            self.token is None
-            or self.last_refresh_time is None
-            or time.time() - self.last_refresh_time > AZURE_TOKEN_REFRESH_INTERVAL
-        ):
-            self.last_refresh_time = time.time()
-            self.token = self.credential.get_token(self.token_scope.value).token
-
-        return self.token
-
-
-class _UAIAPITokenManager(_APITokenManager):
+class _WorkspaceConnectionTokenManager(_APITokenManager):
```

Comment:
Do we need to inherit from _APITokenManager

Reply:
no, I will clean this up in a follow up PR